src/main/java/de/csbdresden/stardist/StarDist2DBase.java

```diff
@@ -88,30 +88,38 @@ protected void export(String outputType, Candidates polygons, int framePosition)
     
     protected void exportROIs(Candidates polygons, int framePosition) {
         if (roiManager == null) {
-            IJ.run("ROI Manager...", "");
-            roiManager = RoiManager.getInstance();
+            roiManager = RoiManager.getRoiManager();
```

Comment:
I'm confused by all the different ways of getting the `RoiManager` (`getRoiManager()`, `getInstance()`, `getRawInstance()`, `getInstance2()`).

Does it matter in practice? I see that some are meant to be compatible with batch mode macro?

Reply:
The most recent addition was `RoiManager.getRoiManager()` which is supposed to replace 
```
RoiManager rm = RoiManager.getInstance()
if (rm == null) rm = new RoiManager()
```
I would deprecate the others personally and stick to `getRoiManager()`

Reply:
Hmm, from [looking at the code](https://github.com/imagej/ImageJA/blob/b78e7ca95ee93609fef29496e3d9e557314ef2e2/src/main/java/ij/plugin/frame/RoiManager.java#L1898-L1931) it doesn't do exactly the same, because `getInstance()` will do something special when running a batch mode macro (no idea if this is something to be worried about).

Reply:
Thanks for looking into it! 
Looking further, I see that in the Interpreter class it makes no difference. I can vouch for that. We have been using the RoiManager in easily 10 or more IJ1 Plugins and always called it the way I showed you. Worked fine in batch mode as well.